src/utils/getResponse.ts

```diff
@@ -90,6 +86,11 @@ export const getResponse = async <
       }
 
       if (mockedResponse && mockedResponse.once) {
+        // Skip a handler if it has been already used for a one-time response.
+        if (requestHandler.shouldSkip) {
```

Comment:
What about moving this to line 84?
Reasoning,  it would also catch other skip conditions in the future.

```ts
if (!mockedResponse || requestHandler.shouldSkip) {
        return acc
}
```

Reply:
Agreed.